orders/models.py

```diff
@@ -0,0 +1,62 @@
+from django.db       import models
+
+from users.models    import User
+from products.models import Product
+
+class Order(models.Model):
+    user         = models.ForeignKey(User, on_delete=models.CASCADE)
+    order_status = models.ForeignKey('OrderStatus', on_delete=models.CASCADE)
+    updated_at   = models.DateTimeField(auto_now=True)
+    created_at   = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        db_table = 'orders'
+
+class OrderStatus(models.Model):
+    order_status_description = models.CharField(max_length=100)
+    updated_at               = models.DateTimeField(auto_now=True)
+    created_at               = models.DateTimeField(auto_now_add=True)    
+
+    class Meta:
+        db_table = 'order_status'
+
+class OrderItem(models.Model):
+    quantity          = models.IntegerField()
+    product           = models.ForeignKey(Product, on_delete=models.CASCADE)
+    order             = models.ForeignKey(Order, on_delete=models.CASCADE)
+    order_item_status = models.ForeignKey('OrderItemStatus',on_delete=models.CASCADE)
+    updated_at        = models.DateTimeField(auto_now=True)
+    created_at        = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        db_table = 'order_items'
+
+class OrderItemStatus(models.Model):
+    order_item_status_description = models.CharField(max_length=100)
+    updated_at                    = models.DateTimeField(auto_now=True)
+    created_at                    = models.DateTimeField(auto_now_add=True)    
```

Comment:
변수명은 간결하고 명확하게 만들어주세요~
```suggestion

class OrderItemStatus(models.Model):
    status = models.CharField(max_length=100)
    updated_at                    = models.DateTimeField(auto_now=True)
    created_at                    = models.DateTimeField(auto_now_add=True)    
```

Reply:
변수명 수정하였고 다른 테이블의 변수명도 명확하게 추가 수정했습니다.